java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/ProfileActivity.java

```diff
@@ -34,11 +34,52 @@
 import android.widget.AdapterView.OnItemSelectedListener;
 
 public class ProfileActivity extends Activity {
+	private HealthVaultApp mService;
+	private HealthVaultClient mHVClient;
+	private Record mCurrentRecord;
+
 	@Override
 	protected void onCreate(@Nullable Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.profile);
+		mService = HealthVaultApp.getInstance();
+		mHVClient = new HealthVaultClient();
 
 		setTitle("Profile Sample");
 	}
+
+	private void populateProfile() {
+		final EditText firstName = (EditText) findViewById(R.id.firstNameText);
+		final EditText secondtName = (EditText) findViewById(R.id.firstNameText);
+		firstName.setText("hatim");
+	}
+
+	@Override
+	protected void onResume() {
+		super.onResume();
+		mHVClient.start();
```

Comment:
Understood.  There is only 4 activities with not much code for the samples

---
In reply to: [118756666](https://github.com/Microsoft/healthvault-samples/pull/13#discussion_r118756666) [](ancestors = 118756666)